src/test/java/io/vertx/test/codegen/converter/TestDataObject.java

```diff
@@ -11,1190 +11,1326 @@
-    return this;
-  }
-
-  public Map<String, AggregatedDataObject> getKeyedDataObjectValues() {
-    return keyedDataObjectValues;
-  }
-
-  public TestDataObject addKeyedDataObjectValue(String key, AggregatedDataObject value) {
-    keyedDataObjectValues.put(key, value);
-    return this;
-  }
-
-  public Map<String, Buffer> getKeyedBufferValues() {
-    return keyedBufferValues;
-  }
-
-  public TestDataObject addKeyedBufferValue(String key, Buffer value) {
-    keyedBufferValues.put(key, value);
-    return this;
-  }
-
-  public Map<String, JsonObject> getKeyedJsonObjectValues() {
-    return keyedJsonObjectValues;
-  }
-
-  public TestDataObject addKeyedJsonObjectValue(String key, JsonObject value) {
-    keyedJsonObjectValues.put(key, value);
-    return this;
-  }
-
-  public Map<String, JsonArray> getKeyedJsonArrayValues() {
-    return keyedJsonArrayValues;
-  }
-
-  public TestDataObject addKeyedJsonArrayValue(String key, JsonArray value) {
-    keyedJsonArrayValues.put(key, value);
-    return this;
-  }
-
-  public Map<String, TimeUnit> getKeyedEnumValues() {
-    return keyedEnumValues;
-  }
-
-  public TestDataObject addKeyedEnumValue(String key, TimeUnit value) {
-    keyedEnumValues.put(key, value);
-    return this;
-  }
-
-  public Map<String, ZonedDateTime> getKeyedDateTimeValues() {
-    return keyedDateTimeValues;
-  }
-
-  public TestDataObject addKeyedDateTimeValue(String key, ZonedDateTime value) {
-    this.keyedDateTimeValues.put(key, value);
-    return this;
-  }
-
-  public Map<String, URI> getKeyedUriValues() {
-    return keyedUriValues;
-  }
-
-  public TestDataObject addKeyedUriValue(String key, URI value) {
-    this.keyedUriValues.put(key, value);
-    return this;
-  }
-
-  public Map<String, Object> getKeyedObjectValues() {
-    return keyedObjectValues;
-  }
-
-  public TestDataObject addKeyedObjectValue(String key, Object value) {
-    keyedObjectValues.put(key, value);
-    return this;
-  }
-
-  public List<NoConverterDataObject> getNotConvertibleDataObjects() {
-    return notConvertibleDataObjects;
-  }
-
-  public TestDataObject setNotConvertibleDataObjects(List<NoConverterDataObject> notConvertibleDataObjects) {
-    this.notConvertibleDataObjects = notConvertibleDataObjects;
-    return this;
-  }
-
-  public Set<NoConverterDataObject> getNotConvertibleDataObjectSet() {
-    return notConvertibleDataObjectSet;
-  }
-
-  public TestDataObject setNotConvertibleDataObjectSet(Set<NoConverterDataObject> notConvertibleDataObjectSet) {
-    this.notConvertibleDataObjectSet = notConvertibleDataObjectSet;
-    return this;
-  }
-
-  public Map<String, NoConverterDataObject> getNotConvertibleDataObjectMap() {
-    return notConvertibleDataObjectMap;
-  }
-
-  public TestDataObject setNotConvertibleDataObjectMap(Map<String, NoConverterDataObject> notConvertibleDataObjectMap) {
-    this.notConvertibleDataObjectMap = notConvertibleDataObjectMap;
-    return this;
-  }
+	@Mapper
```

Comment:
can you avoid reformating this code entirely and just add what is necessary ?

Reply:
It should be better now